### DISCUSSION TITLE
Student benefits approved but not active

### DISCUSSION THREAD

**Discussion by dusansel** [community/community#62089] [GitHub Education]
### Hi everyone,

I got the message that
Approved Submitted Jul 23, 2023
Congrats! Your request for [@dusansel](https://github.com/dusansel) was approved and your benefits are now available.

But still I don't have active benefits.
I can see the message:
Hi, dusansel! You were last verified as a student on Jul 23, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

When academic benefits will be active?

**Comment by NafiRahim** [comment]
Hii, @dusansel after applying to the student pack, first they say you are approved and your benefit will be available soon. I had to wait for 74 hr then I got my benefit activated. There is a waiting period between verification and access to benefits such as Copilot, GitHub Pro, etc. Once they are active in your account you will receive an e-mail in your registered mail which you gave during the application saying Thank you for joining GitHub Global Campus. 


So, don't worry since you are already approved as a student, your benefit will be available soon.
